dxr/plugins/python/indexers.py

```diff
@@ -1,7 +1,8 @@
 import ast
 import token
 import tokenize
-from StringIO import StringIO
+from os.path import islink
+from cStringIO import StringIO
```

Comment:
I switched out the import here since I happened to see it. Should I back this out since it's not related?